dockerclient.go

```diff
@@ -15,6 +15,8 @@ import (
 	"strings"
 	"sync/atomic"
 	"time"
+
+	"github.com/docker/docker/api/stats"
```

Comment:
No way, we should never import docker packages directly.

The main purpose of starting this library was to be entirely client side (everything over HTTP, no docker internal import). This can break the library anytime.